hsengine/game.h

```diff
@@ -485,10 +485,10 @@ public slots:
      * board control value function.
      */
     struct BoardControlScore {
-        static const float TOUGH_MINIONS_MULT = 2.5; //!< Multiplier for the "tough minions" factor
-        static const float HAND_MULT = 1;            //!< Multiplier for the "hand" factor
-        static const float MINIONS_MULT = 1.75;      //!< Multiplier for the "minions number" factor
-        static const float SITUATION_MULT = 3.75;    //!< Multiplier for the "situation" factor
+        static constexpr const float TOUGH_MINIONS_MULT = 2.5; //!< Multiplier for the "tough minions" factor
```

Comment:
Is there a specific reason why you did not use the Q_DECL_CONSTEXPR macro from Qt to only insert the constexpr keyword when the compiler supports it? (src: https://woboq.com/blog/cpp11-in-qt5.html)